utils/address.go

```diff
@@ -17,7 +18,7 @@ func ParseCLIAddress(clientCtx client.Context, address string) (string, error) {
 		// empty address --> address = creator
 		address = clientCtx.GetFromAddress().String()
 	} else {
-		if IsBech32Address(address) {
+		if IsBech32Address(address) || address == commontypes.EMPTY_PROVIDER {
```

Comment:
should use
import "github.com/lavanet/lava/x/dualstaking/types"
types.EMPTY_PROVIDER